src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -732,6 +737,8 @@ private async Task SendRequestContentAsync(HttpRequestMessage request, HttpConte
 
             // Flush any content that might still be buffered.
             await FlushAsync().ConfigureAwait(false);
+
+            if (NetEventSource.IsEnabled) Trace("Finished sending request content.");
```

Comment:
Are there cases where this won't match up with the "request fully sent" logging above? It seems like logging the 'completion of request content' and the 'completion of the whole request' is redundant. If there are cases where this is useful though, I'm not against it.

Reply:
1. For cases where request doesn't have Expect Continue header, we will send the body right away, and start to read response header. (if we have Expect Continue header, we will need to read response header first, then determine if will send body).
2. The "Request is fully sent." is after we finished readying response header, and make sure request is fully sent.

This log is helpful to determine if anything unusual happens during "send request content", or read "response header".

Reply:
Okay, in that case this seems fine.